### PR TITLE
Support OPA/REGO policy

### PR DESCRIPTION
**Pull request by afdesk**

### CODE REVIEW

**Comment by jerbia** [comment]
@afdesk can you look at the build issue?

**Comment by afdesk** [comment]
> 
> 
> @afdesk can you look at the build issue?

sure, I'm doing it right now.

**Comment by afdesk** [comment]
@jerbia  @krol3 I've changed go version from from 1.12 to 1.15.
Is it OK?

File: README.md [hunk 1]
```diff
@@ -191,7 +191,18 @@ Ignore-Registry | Optional: comma separated list of registries that will be igno
 Ignore-Image-Name |  Optional: list of comma separated images that will be ignored by the integration
 Aggregate-Issues-Number | Optional: Aggregate multiple scans into one ticket/message | Numeric number. Default is 1
 Aggregate-Issues-Timeout| Optional: Aggregate multiple scans over period of time into one ticket/message | Xs (X number of seconds), Xm (X number of minutes), xH (X number of hours)
+Policy_OPA | Optional: a list of files with OPA/REGO policies | opap.rego
```
Comment (jerbia) [inline README.md:194]: @afdesk where should I put the rego file???

**Review by jerbia — APPROVED** [review]

**Pull request merged**